data.go

```diff
@@ -33,6 +41,7 @@ func ToIngressifyRule(il *v1beta1.IngressList) []IngressifyRule {
 				ir.Path = path.Path
 				ir.ServiceName = path.Backend.ServiceName
 				ir.ServicePort = path.Backend.ServicePort.IntVal
+				ir.Hash = hash(ing.Namespace + path.Backend.ServiceName + ir.Host + ir.Path)
```

Comment:
Do you think we should also mixin the ingress name here :)

Reply:
yeah, im putting the `ing.Name` as well.